src/os/FuseStore.cc

```diff
@@ -420,11 +434,19 @@ static int os_readdir(const char *path,
   switch (t) {
   case FN_ROOT:
     {
+#if FUSE_VERSION >= FUSE_MAKE_VERSION(3, 0)
+      filler(buf, "type", NULL, 0, flags);
+#else
       filler(buf, "type", NULL, 0);
+#endif
```

Comment:
it's better to define a helper function for filer.

Reply:
Sure, will add one.